README.md

```diff
@@ -81,34 +78,48 @@ GH_TOKEN=${GH_TOKEN}
 # Branches
 TMS_BRANCH=main
 ED_BRANCH=main
+RULE_EXECUTER_BRANCH=main
```

Comment:
OK, so sure, the thing we are compiling is the rule executor, but the result is the rule _processor_ and it helps if this is unique if we are also then adding the other rule processors. Why are we rolling back this change?